scripts/main/photo.js

```diff
@@ -812,7 +812,8 @@ photo.editTags = function (photoIDs) {
 			<div class="input-group stacked"><input class='text' name='tags' type='text' minlength='1'></div>
 			<div class='input-group compact-inverse'>
 				<label for="override">${lychee.locale["OVERRIDE"]}</label>
-				<input type='checkbox' id='override' name='override' />
+				<input type='checkbox' id='tag_dialog_override_input' name='override' />
+				<p>${lychee.locale["TAGS_OVERRIDE_INFO"]}</p>
```

Comment:
Don't inject text into HTML code this way. Have a look at the other dialogs how to do it. Use the `.textContent` property of the paragraph to actually set the text.

Reply:
Ah, and basically this also holds for the the label. I don't know how I could miss that in the first place.